core/trino-server-rpm/src/test/java/io/trino/server/rpm/ServerIT.java

```diff
@@ -102,9 +102,12 @@ private static void testServer(String rpmHostPath, String expectedJavaVersion)
                 "yum localinstall -q -y " + rpm + "\n" +
                 // create Hive catalog file
                 "mkdir /etc/trino/catalog\n" +
+                "echo CONFIG_ENV[HMS_PORT]=9083 >> /etc/trino/env.sh\n" +
+                "echo CONFIG_ENV[NODE_ID]=test-node >> /etc/trino/env.sh\n" +
```

Comment:
make the "test-node" value more unique eg "test-node-id-injected-via-env"

Reply:
Done